apps/userdashboard/assets/js/a4_candy_userdashboard/ModerationNotification.jsx

```diff
@@ -255,8 +257,8 @@ export const ModerationNotification = (props) => {
   }
 
   return (
-    <>
-      <li className="list-item">
+    <li className="list-item border-0 p-0">
+      <div className="list-item">
```

Comment:
is this div still needed and should it use the same class `list-item` as its parent?

Reply:
the div is needed to ensure that the mod statement and comment are in the same border box but the comment box for the statement is external as in the designs but you are right, double use of list-item is not great, i'll go back and reconsider